border-center-for-journalists/border-hub-website
# Popup Images

#### What does this PR do?
Adds a pop-up to the images on a notice.
#### Where should the reviewer start from?
`package.json`
`gatsby-config.js`
`src/components/lightbox/index.js`
`src/components/notice/textContent.js`
#### How should this be tested?
* Enter a notice
* Tap on an image from the content
* The page should show the modal with the image
* <kbd>ESC</kbd> , Click on `X` or Click outside the modal should close it.
#### Any background context you want to provide?
* Added react-modal-responsive library
* Changed the Image return in HtmlSerializer
  * TextContent parses html before setting up `dangerouslyInnerHtml`to retrieve TextContent or ImageModals 
* Added ImageModal component
#### Questions?
